docs/source/getstarted.rst

```diff
@@ -31,8 +31,8 @@ Starting Up the SDK
 2. Open the Cozmo app on the phone. Make sure Cozmo is on and connected to the app via WiFi.
 3. Tap on the gear icon at the top right corner to open the Settings menu.
 4. Swipe left to show the Cozmo SDK option and tap the Enable SDK button.
-5. On the computer, open a Terminal (macOS/Linux) / Command Prompt (Windows) window. Type ``cd cozmo_sdk_examples`` where *cozmo_sdk_examples* is the name of the directory you extracted the SDK examples into and press Enter. The SDK example scripts can be downloaded from the :doc:`Downloads page </downloads>`.
-6. Now you can run any of the example programs.
+5. Make sure the SDK examples are downloaded from the :doc:`Downloads page </downloads>`.
+6. On the computer, open Terminal (macOS/Linux) or Command Prompt (Windows) and navigate to the folder containing that program.
```

Comment:
It looks like we are losing the following text: "Type ``cd cozmo_sdk_examples`` where *cozmo_sdk_examples* is the name of the directory you extracted the SDK examples into and press Enter."

Instead we are telling users to "navigate to the folder containing that program." But we aren't telling them more clearly where to go.

In addition, at this point we want to navigate into the example programs in general, so the new text specifically referring to "that program" isn't quite right.